health/services.py

```diff
@@ -40,13 +40,20 @@ def fn(app):
         client = app.config["HTTPCLIENT"]
         registry_url = _compute_internal_endpoint(app, endpoint)
         try:
-            status_code = client.get(registry_url, verify=False, timeout=2).status_code
+            status_code = client.get(
+                registry_url, verify=False, timeout=app.config["GUNICORN_HEALTH_CHECK_TIMEOUT"]
```

Comment:
you'll want to do `app.config.get("GUNICORN_HEALTH_CHECK_TIMEOUT", 2)` instead

Reply:
good point, updated, thanks!